app/src/main/java/co/nimblehq/alpine/sample/extension/ActivityExt.kt

```diff
@@ -27,3 +38,31 @@ fun Activity.getStatusBarHeight(): Int {
     val resourceId = resources.getIdentifier("status_bar_height", "dimen", "android")
     return if (resourceId > 0) resources.getDimensionPixelSize(resourceId) else 0
 }
+
+fun Activity.getMediaOutputDirectory(): File {
+    val mediaDir = applicationContext.externalMediaDirs.firstOrNull()?.let {
+        File(it, applicationContext.resources.getString(R.string.app_name)).apply { mkdirs() }
+    }
+    return if (mediaDir != null && mediaDir.exists()) {
+        mediaDir
+    } else {
+        applicationContext.filesDir
+    }
+}
+
+fun Activity.notifyOnNewPhotoAdded(savedUri: Uri) {
```

Comment:
Not sure we need this method, we don't seem to be observing the broadcast anywhere 🤔

Reply:
Yeah, they are redundant and I forgot to delete them when doing copy/paste 😆